manage.py

```diff
@@ -251,6 +250,32 @@ def demo_db():
     for i in range(5):
         add_passover(timestamp=now + timedelta(minutes=i*10))
 
+    print("Database has been seeded with demo data.")
+
+
+def import_commands():
+    """Imports commands from the CommandDocs.txt file in ex2_ground_station_software.
+    """
+    try:
+        with open("ex2_ground_station_software/CommandDocs.txt", 'r') as f:
```

Comment:
One thing that may need to be considered in the future is that gs_software commands need to specify the 'server' that these commands go, which I believe CommandDocs.txt doesn't specify.

For example, the command `time_management.get_eps_time` listed in the txt file should be called with `eps.time_management.get_eps_time` to send the command to the correct component. I'm not exactly sure how these are determined (came across this while testing gs_software's cli), best to check with Ryan and the software team to figure out which commands belong to which server ('OBC' or 'EPS'). 

Other files to check out in gs_software are `src/groundStation/system.py` for a more up-to-date list of all working commands in the form of Python dictionaries and `src/groundStation/commandParser.py` to see how a command string is parsed and validated according to `system.py`

Reply:
For now, take a look at how commands are validated in [LiveCommands.jsx here](https://github.com/AlbertaSat/ex2_ground_station_website/blob/107e54a4b3bb28f3a4d30615979291db146d64ca/groundstation/static/js/components/LiveCommands.jsx#L121) and make it ignore the first 4 characters of a command (ie. the server). I believe that ex2_ground_station_software also does not validate whether or not a command goes to the right server.